executor.go

```diff
@@ -197,6 +199,12 @@ func (e *executor) executeCall(ctx context.Context, index string, c *pql.Call, s
 	case "TopN":
 		e.Holder.Stats.CountWithCustomTags(c.Name, 1, 1.0, []string{indexTag})
 		return e.executeTopN(ctx, index, c, shards, opt)
+	case "Rows":
```

Comment:
I like the concept of Column returning all the details about a specific column (rows and attributes). I think the label breaks down when we are trying to interact with all/some rows contained in a field.  I feel we need an additional term for dealing with the field based issues.  Perhaps introduce some ...Info based terms ie FieldInfo, ColumnInfo,RowInfo.  Still feels hackish, hoping this will trigger some synapse action elsewhere.